browsers/browsers.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+
+  "definitions": {
+
+    "browser_statement": {
+      "type": "object",
+      "properties": {
+        "releases": {
+          "type": "object",
+          "additionalProperties": { "$ref": "#/definitions/release_statement" }
+        }
+      },
+      "required": ["releases"],
+      "additionalProperties": false
+    },
+
+    "release_statement": {
+      "type": "object",
+      "properties": {
+        "release_date": { "type": "string", "format": "date" },
+        "release_notes": { "type": "string", "format": "uri" },
+        "status": {
+          "type": "string",
+          "enum": ["retired", "current", "beta", "nightly", "esr", "planned"]
+        }
+      },
+      "required": ["status"]
+    }
+
+  },
+
+  "type": "object",
+    "properties": {
+      "webview_android": { "$ref": "#/definitions/browser_statement" },
+      "chrome": { "$ref": "#/definitions/browser_statement" },
+      "chrome_android": { "$ref": "#/definitions/browser_statement" },
+      "edge": { "$ref": "#/definitions/browser_statement" },
+      "edge_mobile": { "$ref": "#/definitions/browser_statement" },
+      "firefox": { "$ref": "#/definitions/browser_statement" },
+      "firefox_android": { "$ref": "#/definitions/browser_statement" },
+      "ie_mobile": { "$ref": "#/definitions/browser_statement" },
+      "ie": { "$ref": "#/definitions/browser_statement" },
+      "nodejs": { "$ref": "#/definitions/browser_statement" },
+      "opera": { "$ref": "#/definitions/browser_statement" },
+      "opera_android": { "$ref": "#/definitions/browser_statement" },
+      "safari": { "$ref": "#/definitions/browser_statement" },
+      "safari_ios": { "$ref": "#/definitions/browser_statement" }
+    },
+    "additionalProperties": false
```

Comment:
As we are limiting the name of the browsers, do we want to add "samsung_internet_android" to welcome participation?

Reply:
This list is a copy of https://github.com/mdn/browser-compat-data/blob/master/compat-data.schema.json#L66. Adding a new browser should be discussed in a new issue and in the weekly meetings.

Reply:
FIne for me.

Reply:
I' would like to see Samsung added, FWIW, as they are being friendly, and doing good evangelism currently

Reply:
I also think we should add it, just not as part of this PR :)